requirements.txt

```diff
@@ -0,0 +1,9 @@
+analysis-runner==2.14.0
```

Comment:
Do you need the `analysis-runner` dependency?

Reply:
not for this script, but the analysis-runner package is used for the hail wrapper script which will call this. The requirements file isn't strictly limited to this PR, but I could make it this way

Reply:
If you could strip it down to the minimum set, I think that would be helpful, so we can spot whether all requirements are really needed.